Add-on signing table updates (#1897)
mozilla/extension-workshop: src/content/documentation/publish/signing-and-distribution-overview.md

| Signing method     | Supported distribution channel(s) | 
| ------------------------------------- | ------------------- | 
| Web upload via the [AMO Developer Hub](https://addons.mozilla.org/developers/) | Public listing on AMO or self-distribution (unlisted). | 

> | Upload using [`web-ext sign`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-sign) | **Version 8 and higher**: Public listing on AMO or self-distribution (unlisted). <br /><br />**Version 7 and lower**: Initial submissions only provide for self-distributed (unlisted) extensions. Updates can be for extensions listed on AMO or self-distributed (unlisted). |

Reviewer: ```suggestion
| Upload using [`web-ext sign`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-sign) | **Version 8 and higher**: Initial submission and updates for extensions listed on AMO or self-distributed (unlisted). <br /><br />**Version 7 and lower**: Initial submissions for self-distributed (unlisted) extensions. Updates for extensions listed on AMO or self-distributed (unlisted). |
```

> | Upload using [web-ext sign](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-sign) | <strong>Version 8 and higher:</strong> Public listing on AMO or self-distribution (unlisted). <br /><br /><strong>Version 7 and lower:</strong> New add-on submissions can only be submitted as self-distributed (unlisted) extensions. Subsequent add-on updates can be listed on AMO or self-distributed (unlisted). | 

Reviewer: Unless necessary stick to markdown?
```suggestion
| Upload using [`web-ext sign`](https://extensionworkshop.com/documentation/develop/web-ext-command-reference/#web-ext-sign) | **Version 8 and higher**: Public listing on AMO or self-distribution (unlisted). <br /><br />**Version 7 and lower**: Initial submissions only provide for self-distributed (unlisted) extensions. Updates can be for extensions listed on AMO or self-distributed (unlisted). | 
```

Author: I'm not sure why I used HTML instead of Markdown here. I'll go with your suggestion.

> | The AMO [Add-on Create API (V5)](https://mozilla.github.io/addons-server/topics/api/addons.html#create) and [Version Create API (V5)](https://mozilla.github.io/addons-server/topics/api/addons.html#version-create) | Public listing on AMO or self-distribution (unlisted). |

Reviewer: ```suggestion
| The AMO [Add-on Create API (V5)](https://mozilla.github.io/addons-server/topics/api/addons.html#create) and [Version Create API (V5)](https://mozilla.github.io/addons-server/topics/api/addons.html#version-create) | Initial submission and updates for extensions listed on AMO or self-distributed (unlisted). |
```

> | AMO's [Add-on Create API (V5)](https://mozilla.github.io/addons-server/topics/api/addons.html#create) and [Version Create API (V5)](https://mozilla.github.io/addons-server/topics/api/addons.html#version-create) | Public listing on AMO or self-distribution (unlisted). |

Reviewer: ```suggestion
| The AMO [Add-on Create API (V5)](https://mozilla.github.io/addons-server/topics/api/addons.html#create) and [Version Create API (V5)](https://mozilla.github.io/addons-server/topics/api/addons.html#version-create) | Public listing on AMO or self-distribution (unlisted). |
```

> | The AMO [Signing API (V4)](https://addons-server.readthedocs.io/en/latest/topics/api/v4_frozen/signing.html) | Initial submissions only provide for self-distributed (unlisted) extensions. Updates can be to extensions publicly listed on AMO or self-distributed (unlisted). |

Reviewer: ```suggestion
| The AMO [Signing API (V4)](https://addons-server.readthedocs.io/en/latest/topics/api/v4_frozen/signing.html) | Initial submissions for self-distributed (unlisted) extensions. Updates to extensions publicly listed on AMO or self-distributed (unlisted). |
```

> | AMO's [Signing API (V4)](https://addons-server.readthedocs.io/en/latest/topics/api/v4_frozen/signing.html) | New add-on submissions can only be submitted as self-distributed (unlisted) extensions. Subsequent add-on updates can be publicly listed on AMO or self-distributed (unlisted). |

Reviewer: ```suggestion
| The AMO [Signing API (V4)](https://addons-server.readthedocs.io/en/latest/topics/api/v4_frozen/signing.html) | Initial submissions only provide for self-distributed (unlisted) extensions. Updates can be to extensions publicly listed on AMO or self-distributed (unlisted). |
```
